Doc/extending/extending.rst

```diff
@@ -343,6 +346,16 @@ accept a third ``PyObject *`` parameter which will be a dictionary of keywords.
 Use :c:func:`PyArg_ParseTupleAndKeywords` to parse the arguments to such a
 function.
 
+The fourth entry is the function's docstring. Note the ``"system(command)\n--\n\n"``
```

Comment:
Please limit lines to 80 characters (https://devguide.python.org/documenting/#use-of-whitespace).